gen

```diff
@@ -20,12 +20,19 @@
 
 version="Generate (gen) version: 1.0"
 
+ext=$(echo ${1##*.})
+templates=templates/template.$ext
+if [[ -f $templates ]]; then
+  template=$(cat "$templates")
+fi
+
 touchpy() {
 cat << EOF
 def main():
 
 if __name__ == '__main__':
     main()
+${template}
```

Comment:
template or templates here?

Reply:
That would be `template`. It's the template.ext file defined at the top of the script, in the templates folder.